src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -2363,6 +2375,14 @@ public static bool IsLabelContext(this SyntaxTree syntaxTree, int position, Canc
                 return true;
             }
 
+            // using await ( |
+            if (token.IsKind(SyntaxKind.OpenParenToken) &&
```

Comment:
we can just update the previous test to see if the open paren parent is a UsingStatementSyntax.